README.md

```diff
@@ -1 +1,112 @@
-# Personal_Portfolio
\ No newline at end of file
+# Zohal Mirkhel's Portfolio
+
+This portfolio showcases the projects and skills of Zohal Mirkhel, a web developer specializing in HTML, CSS, JavaScript, Java, SQL, WordPress, and Bootstrap. The portfolio is designed with a modern, minimalist aesthetic and features a dark theme.
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Features](#features)
+- [Technologies Used](#technologies-used)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Contact](#contact)
+
+## Introduction
+
+Welcome to my portfolio! This website highlights my journey as a web developer, showcasing my skills, projects, and professional experience. It is designed to provide a comprehensive view of my capabilities and accomplishments.
+
+## Features
+
+- **Responsive Design**: The portfolio is fully responsive and adapts to different screen sizes and devices.
+- **Navigation Bar**: Includes links to Home, About, Work, and Contact sections.
+- **Contact Information**: Includes email, phone number, address, and working hours.
+- **Project Showcase**: Highlights various projects with descriptions and links to live sites and repositories.
+- **Skills and Experience**: Details my professional background and technical skills.
+- **Contact Form**: Allows visitors to contact me directly through the website.
+
+
+## Technologies Used
+
+- **HTML**: Structure and content of the web pages.
+- **CSS**: Styling and layout of the web pages, including a dark theme and responsive design.
+- **JavaScript**: Interactive elements and form validation.
+- **FontAwesome**: Icons used throughout the site.
+- **EmailJS**: For contact form integration.
+- **Formspree**: For contact form integration.
+
+
+
+## Live Demo
+
+![alt text](images/image.png)
+You can view a live demo of the project [here](https://zohalmirkhel.github.io/Personal_Portfolio/)
+
+
+## Installation
+
+To set up this project locally, follow these steps:
+
+
+1. **Clone the repository**:
+   ```bash
+   git clone git@github.com:ZohalMirkhel/Personal_Portfolio.git
+   ```
+2. **Navigate to the project directory**:
+   ```bash
+   cd portfolio
+   ```
+
+3. **Open the `index.html` file in your browser**:
+   ```bash
+   open index.html
+   ```
+
+4. **View the source code:**
+If you want to explore the source code, open the project in your preferred code editor (e.g., VSCode):
+   ```bash
+   code .
+   ```
+
+
+## Technologies Used
+
+* HTML
+* CSS
+* JavaScript
+* EmailJS
+
+Note: No additional installations or dependencies are required for the JavaScript to work properly. The code runs directly in the browser.
+
+## Contribution
+1. **Hadisa Zahirzai**.
+2. **Zabeena Sherzoie**.
+3. **Safia Iqbal**.
```

Comment:
- Well done, your README.md file is very great  just implement these 2 changes:
  - At least add one social profile link for the UI/UX Designers besides their name.
  - Add the link of the Figma Design on your README